README.md

```diff
@@ -1 +1,25 @@
-# a1-decide
\ No newline at end of file
+# a1-decide
```

Comment:
We can call the program for **Launch Interceptor Program** or something similar (and more describing)

![bild](https://user-images.githubusercontent.com/70259607/151852707-dc9c7ece-95d2-4418-97b1-5cc671b9a1cc.png)